src/Symfony/Component/Translation/Extractor/PhpExtractor.php

```diff
@@ -135,7 +133,7 @@ class PhpExtractor extends AbstractFileExtractor implements ExtractorInterface
     /**
      * {@inheritdoc}
      */
-    public function extract(string|iterable $resource, MessageCatalogue $catalog)
+    public function extract(string | iterable $resource, MessageCatalogue $catalog)
```

Comment:
should be reverted (fabbot has a false positive about spaces in union type)

Reply:
reverted in f7663551312b74366de018a963b1fd98091631da